script/cmd/DeployTokenCallbackPlugin

```diff
@@ -0,0 +1 @@
983030160408a01526109cb565b90518487830301848801526109cb565b940151918184860301608085015282519081865280860181808460051b8901019501936000975b8489106107cf5787870388f35b9091929394958480610805838686600196030188526040838c5163ffffffff60e01b8151168452015191818582015201906109cb565b980194019801979190949392946107c2565b50346101255760403660031901126101255780359060ff8216820361012557602435906001600160401b0382116101255761085491369101610949565b5050610bf0565b823461012557602036600319011261012557359063ffffffff60e01b82168092036101255760209163043140ed60e01b811490811561089c575b5015158152f35b6301ffc9a760e01b14905083610895565b82346101255760c0366003190112610125576108c761091d565b506108d0610933565b506044356001600160a01b03811603610125576001600160401b0390608435828111610125576109039036908301610949565b505060a4359182116101255761091b91369101610949565b005b600435906001600160a01b038216820361012557565b602435906001600160a01b038216820361012557565b9181601f84011215610125578235916001600160401b038311610125576020838186019501011161012557565b9060806003198301126101255760043560ff8116810361012557916024356001600160a01b0381168103610125579160443591606435906001600160401b038211610125576109c791600401610949565b9091565b919082519283825260005b8481106109f7575050826000602080949584010152601f8019910116010190565b6020818301810151848301820152016109d6565b3461012557600319606036820112610125576004359060ff8216820361012557602435906001600160401b03821161012557610120913603011215610bf057600080fd5b34610125576020366003190112610125576004356001600160401b0381116101255761091b903690600401610949565b60405190604082018281106001600160401b03821117610a9e57604052565b634e487b7160e01b600052604160045260246000fd5b60405190606082018281106001600160401b03821117610a9e57604052565b6040519060a082018281106001600160401b03821117610a9e57604052565b9181601f84011215610125578235916001600160401b038311610125576020808501948460051b01011161012557565b90815180825260208080930193019160005b828110610b42575050505090565b83516001600160e01b03191685529381019392810192600101610b34565b80516005811015610b86576040918291845260ff60208201511660208501520151910152565b634e487b7160e01b600052602160045260246000fd5b90815180825260208080930193019160005b828110610bbc575050505090565b90919293826080600192610be483895163ffffffff60e01b8151168452015184830190610b60565b01950193929101610bae565b6040516384b9b37960e01b81526000356001600160e01b031916600482015260ff9091166024820152604490fd5b60405190608082018281106001600160401b03821117610a9e57604052606080838181528160208201528160408201520152565b6040519061018082018281106001600160401b03821117610a9e5760405281610160606091828152826020820152826040820152828082015260006080820152600060a08201528260c08201528260e08201528261010082015282610120820152826101408201520152565b805115610ccb5760200190565b634e487b7160e01b600052603260045260246000fd5b805160011015610ccb5760400190565b805160021015610ccb5760600190565b805160031015610ccb576080019056fea26469706673582212205e81eb8270352b3e36936caf6244b9804c7d4651c344dd58c7114f0f93f5902b64736f6c63430008180033 --etherscan-api-key $ETHERSCAN_API_KEY
```

Comment:
Should we remove [80002_run-1725650624_DeployTokenCallbackPlugin]

Reply:
[done](https://github.com/circlefin/buidl-wallet-contracts/pull/12/commits/504fecab677609369782a013eb57d97bf54f6b85)

Reply:
You removed the verify folder one but not the cmd folder

Reply:
ah okay didn't realize we have two